coldfront/plugins/qumulo/utils/acl_allocations.py

```diff
@@ -177,15 +178,49 @@ def set_allocation_attributes(
             value=f"storage-{wustlkey}-{acl_type}",
         )
 
+    @staticmethod
+    def get_allocation_rwro_group_name(access_allocations, rwro):
+        allocation = next(
+            filter(
+                lambda access_allocation: access_allocation.resources.filter(
+                    name=rwro
+                ).exists(),
+                access_allocations,
+            )
+        )
+        return allocation.get_attribute(name="storage_acl_name")
+
+    @staticmethod
+    def reset_allocation_acls(allocation: Allocation, qumulo_api: QumuloAPI):
+        return AclAllocations.set_or_reset_allocation_acls(
+            allocation,
+            qumulo_api,
+            True
+        )
+
     @staticmethod
     def set_allocation_acls(
-        base_allocation: Allocation,
+        allocation: Allocation,
+        qumulo_api: QumuloAPI,
+    ):
+        return AclAllocations.set_or_reset_allocation_acls(
+            allocation,
+            qumulo_api,
+            False
+        )
+
+    @staticmethod
+    def set_or_reset_allocation_acls(
+        allocation: Allocation,
         qumulo_api: QumuloAPI,
+        reset: bool
     ):
-        fs_path = base_allocation.get_attribute("storage_filesystem_path")
+        global logger
+        fs_path = allocation.get_attribute("storage_filesystem_path")
         acl = qumulo_api.rc.fs.get_acl_v2(fs_path)
 
-        access_allocations = AclAllocations.get_access_allocations(base_allocation)
+        access_allocations = AclAllocations.get_access_allocations(allocation)
+        # start: this piece might want to use get_allocation_rwro_group_name() above
```

Comment:
Good idea, I like the reduction of replicated code.  I would go ahead and do that and then remove this comment